File: Data Structures & Algorithms/single-number/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    singleNumber(nums) {
        let set = new Set();
        for (let num of nums){
           if(set.has(num)){
            set.delete(num);
           }else{
            set.add(num);
           }
           
        }
        for(let item of set){
            return item;
        }
    }
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array-ii/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if (nums.length <= 2) return nums.length;

        int L = 2;
        for (int R = 2; R < nums.length; R++) {
            if (nums[R] != nums[L - 2]) {
                nums[L] = nums[R];
                L++;
            }

        }

        return L;
    }
}